simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/samplebuilder/MapperConfig.java

```diff
@@ -0,0 +1,154 @@
+package io.prometheus.client.dropwizard.samplebuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+/**
+ * POJO containing info on how to map a graphite metric to a prometheus one.
+ * Example mapping in yaml format:
+ * <p>
+ * match: test.dispatcher.*.*.*
+ * name: dispatcher_events_total
+ * labels:
+ * action: ${1}
+ * outcome: ${2}_out
+ * processor: ${0}
+ * status: ${1}_${2}
+ * <p>
+ * Dropwizard metrics that match the "match" pattern will be further processed to have a new name and new labels based on this config.
+ */
+public class MapperConfig {
+    // each part of the metric name between dots
+    private static final String METRIC_PART_REGEX = "[a-zA-Z_0-9](-?[a-zA-Z0-9_])+";
+    // Simplified GLOB: we can have "*." at the beginning and "*" only at the end
+    static final String METRIC_GLOB_REGEX = "^(\\*\\.|" + METRIC_PART_REGEX + "\\.)+(\\*|" + METRIC_PART_REGEX + ")$";
+    // Labels validation.
+    private static final String LABEL_REGEX = "^[a-zA-Z_][a-zA-Z0-9_]+$";
```

Comment:
What if there's a replacement in the label or metric name?

Reply:
I'm already formatting metric names, not label names, but if you think it's useful I can do that.

Reply:
Hmm, looks like the Go ones don't do this so you can leave it as-is

Reply:
Cool thank you